<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>772</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>맑은 고딕</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PX4Forensic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dataRefreshButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>데이터 새로 불러오기</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_connected">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>59</y>
      <width>851</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="FTPTab">
     <attribute name="title">
      <string>FTP점검</string>
     </attribute>
     <widget class="QTableWidget" name="ftp_result_tableWidget">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>181</width>
        <height>501</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::DotLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>82</number>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>항목</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>점검 결과</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>T07</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>T10</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>T11</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>T12</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>T31</string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>T38</string>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>T43</string>
       </property>
      </item>
     </widget>
     <widget class="QListWidget" name="ftp_listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>501</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>T07. 암호화</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T10. 파일 및 디렉터리 접근 통제</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T11. 중요 데이터 백업</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T12. 불필요한 정보 노출 최소화</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T31. 중요 파일 무결성 검증</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T38. 로그 생성 및 감사</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T43. 특정 지역 접근 방지</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="ftp_start_pushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>점검 시작</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ftp_result_textEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>381</width>
        <height>331</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;점검 결과&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;선택한 항목&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ftp_selected_textEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="MAVTab">
     <attribute name="title">
      <string>MAVLink점검</string>
     </attribute>
     <widget class="QListWidget" name="mavlink_listWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>501</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>T06. 암호 통신</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T22. 통신 메시지 무결성 검증</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T46. H/W 동작 상태 점검</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T51. 내부 전송 데이터 보호</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T53. 세션 관리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T54. 시간 동기화</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T64. CAN 통신 보안</string>
       </property>
      </item>
     </widget>
     <widget class="QTableWidget" name="mavlink_result_tableWidget">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>181</width>
        <height>501</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::DotLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>82</number>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>항목</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>점검 결과</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>T06</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>T22</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>T46</string>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>T51</string>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>T53</string>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>T54</string>
       </property>
      </item>
      <item row="5" column="1">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>T64</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="mavlink_start_pushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>점검 시작</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;선택한 항목&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="mavlink_selected_textEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;점검 결과&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="mavlink_result_textEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>381</width>
        <height>111</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>320</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;송신된 패킷&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>420</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;수신된 패킷&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="mavlink_recvpacket_textEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>450</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionPORT1">
   <property name="text">
    <string>PORT1</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
